src/ansys/dpf/post/fluid_simulation.py

```diff
@@ -276,6 +285,10 @@ def _get_result(
                 "are mutually exclusive."
             )
 
+        if result_type == _result_type_enum.integrated:
+            # TODO accept only face zones
```

Comment:
For now this is OK, but in the future we could also have integrated results on cells. For example, Heat generation on elements (in W), which would come from a heat volumetric density (W/m^3)

Reply:
Hey @rafacanton, I modified the logic to use a `native_location`. This way when marked as `integrated=True` I only check that the location asked is the same as the native.

Reply:
This makes sense to me, thanks!